sys-kernel/bliss-kernel-bin/bliss-kernel-bin-4.14.170.ebuild

```diff
@@ -0,0 +1,36 @@
+# Copyright 2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit kernel-install
+
+# Variables
+_LV="FC.01"                     # Local Version
+_PLV="${PV}-${_LV}"             # Package Version + Local Version (Module Dir)
+
+# Main
+DESCRIPTION="Precompiled Vanilla Kernel (Kernel Ready-to-Eat [KRE])"
+HOMEPAGE="https://wiki.gentoo.org/wiki/User:Fearedbliss"
+SRC_URI="https://xyinn.org/gentoo/kernels/${_PLV}/kernel-${_PLV}.tar.xz"
+
+RESTRICT="strip"
+LICENSE="GPL-2"
+SLOT="${_PLV}"
+KEYWORDS="-* ~amd64"
+
+# Unset these since 'bliss-kernel' doesn't need them.
+IUSE="-initramfs -test"
```

Comment:
does it really make sense to disable something explicitly? This is not going to enabled unless a user explicitly toggles the flag.

Reply:
In this case it does since after I added `kernel-install` as a dependency of my ebuild, `kernel-install` eclass does an explicit `IUSE="+initramfs test"` which is pulling in additional dependencies for my users for dracut. This is not necessary since my users most likely will be using `bliss-initramfs` for zfs generation. They could always enable initramfs if they want dracut, or install dracut explicitly at their own discretion.

https://github.com/gentoo/gentoo/blob/master/eclass/kernel-install.eclass#L50